docs/modules/data-structures/pages/updating-map-entries.adoc

```diff
@@ -278,14 +280,15 @@ Go::
 [source,go]
 ----
 // error checking is omitted for brevity
-distributedMap, _ := hzClient.GetMap("map")
+ctx := context.Background()
+distributedMap, _ := hzClient.GetMap(ctx, "map")
 entries := []types.Entry{
-	types.NewEntry(1, "John"),
+    types.NewEntry(1, "John"),
 	types.NewEntry(2, "Jack"),
 	types.NewEntry(3, "David"),
 }
-distributedMap.PutAll(entries)
-size, _ := distributedMap.Size()
+distributedMap.PutAll(ctx, entries...)
+size, _ := distributedMap.Size(ctx)
```

Comment:
Not this line, but can you remove 2-3 lines below where it says
`
// error checking is omitted for brevity
distributedMap, _ := hzClient.GetMap("map")
`